.github/workflows/test-deploy.yml

```diff
@@ -0,0 +1,73 @@
+name: Deploy and Configure Morio
```

Comment:
Can we name this file `itest-deploy.yml`, just to differentiate between these integration tests (itest) and other tests.

Reply:
I've changed the name from test->itest